### ISSUE TITLE
frsky telemerty now supported by Arducopter

### ISSUE THREAD

**Issue by Benoit3535** [vizual54/APM-Mavlink-to-FrSky#6]
Reading the arducopter source code, the frsky telemetry is now supported on Serial port 2. (Availaible from version 3.1 -to be checked)
See file config.h and parameters.pde

**Comment by vizual54** [comment]
Yeah I know, didn't know it was merged to master though. I haven't been
following this very closely since I'm working on other projects now that
occupies all of my spare time.

Br
Nils

**Comment by mr-sneezy** [comment]
Is that support just with Pixhawk on serial2, or with APM 2.5 as well (I suspect not) ?
Martin

**Comment by Benoit3535** [comment]
on Pixhawk on serial2 with the official release, on APM you need to activate the feature modifying the firmware and recompiling it. As APM is not as powerfull as Pixhawk you'd better disable some unused features (to save some CPU load) and use it carefully...

**Comment by mr-sneezy** [comment]
Ah, CPU load, the reason to stick with the Arduino Mini or Teensy solutions on APM 2.5 then...
